Gemfile.lock

```diff
@@ -250,25 +245,25 @@ GEM
     method_source (0.9.2)
     mime-types (3.3)
       mime-types-data (~> 3.2015)
-    mime-types-data (3.2019.0904)
+    mime-types-data (3.2019.1009)
     mimemagic (0.3.3)
     mini_magick (4.9.5)
     mini_mime (1.0.2)
     mini_portile2 (2.4.0)
-    minitest (5.12.2)
+    minitest (5.13.0)
     msgpack (1.3.1)
-    mysql2 (0.5.2)
+    mysql2 (0.5.3)
```

Comment:
### [mysql2](https://github.com/brianmario/mysql2)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 0.5.2 → 0.5.3

[_~~change-log~~, ~~source-code~~_]





<details>
<summary>Commits</summary>

A change of **31** commits. See the full changes on [the compare page](https://github.com/brianmario/mysql2/compare/0.5.2...0.5.3).

These are the first 10 commits:
- (bc73887) [Add CentOS on Travis CI. (#989)](https://github.com/brianmario/mysql2/commit/bc73887e67a91bc33c9e69bda15fc079657bccc3)
- (c578718) [Fix code snippet (#1002)](https://github.com/brianmario/mysql2/commit/c578718cb9adc49117358162256420d977381060)
- (98304cb) [Expose windows client authentication (#1018)](https://github.com/brianmario/mysql2/commit/98304cbbc0c9a964c833474a06f55c433ec26432)
- (624409d) [Debian 9 released with MariaDB as the only MySQL variant sin…](https://github.com/brianmario/mysql2/commit/624409d9e493d009653c75f5e550997fa0a93baf)
- (2e5e334) [Remove CHARSETNR_SIZE from generated headers](https://github.com/brianmario/mysql2/commit/2e5e334e1ca6fcb70aaa3450de9cb3af54202d8f)
- (4d0942a) [Warn about missing MySQL encoding mappings](https://github.com/brianmario/mysql2/commit/4d0942a115489b520b3b23ee901f66a1416c0ae2)
- (58b1170) [Fix empty string encoding mappings to NULL](https://github.com/brianmario/mysql2/commit/58b117032ccd95ee8bc65fd2192a0dff38eb062a)
- (a11ac3b) [Add encoding mappings for utf16le and gb18030](https://github.com/brianmario/mysql2/commit/a11ac3b267032d4e57814f1cf8842f029208a1e1)
- (4158af5) [Update encoding tables from MySQL 8.0.16](https://github.com/brianmario/mysql2/commit/4158af5153d874376946cf058b774044f26cd621)
- (b5b18a1) [Update expired MySQL APT signing key](https://github.com/brianmario/mysql2/commit/b5b18a1a26a810ec6e10e9d32d029a3ea00175ea)


</details>